chrome/content/zotero/xpcom/reader.js

```diff
@@ -489,6 +488,20 @@ class ReaderInstance {
 		}
 		// Separator
 		popup.appendChild(this._window.document.createElement('menuseparator'));
+		// Update page number
+		menuitem = this._window.document.createElement('menuitem');
+		menuitem.setAttribute('label', Zotero.getString('pdfReader.updatePageNumber'));
+		menuitem.setAttribute('disabled', readOnly);
+		menuitem.addEventListener('command', () => {
+			let data = {
+				action: 'popupCmd',
+				cmd: 'openPageLabelPopup',
+				standalone,
+				id: currentID
+			};
+			this._postMessage(data);
+		});
+		popup.appendChild(menuitem);
```

Comment:
Without a confirmation on Delete and without Undo, this can't be next to Delete. Undo would be best, but for now let's add a menu separator below.